__tests__/fixtures/action/all_fields_action.output

```diff
@@ -25,3 +25,40 @@ Default test
 
 This action is a `node12` action.
 
+## Usage
```

Comment:
I was not expecting changes in the the test handling acton.yml parsing. Do I miss something?